src/styles/variables.scss

```
@@ -0,0 +1,10 @@
+$square: 100px;
+$font-size: 0.3 * $square;
```

don't do that, just integer
```suggestion
$font-size: px;
```